base/src/ipfixcol.c

```diff
@@ -255,27 +255,39 @@ int main (int argc, char* argv[])
 		MSG_SYSLOG_INIT(PACKAGE);
 		
 		/* and send all following messages to the syslog */
-		if (daemon (1, 0)) {
+		if (daemon(1, 0)) {
 			MSG_ERROR(msg_module, "%s", strerror(errno));
 		}
 	}
 
-	/* check config file */
-	if (config_file == NULL) {
+	/* Check config file */
+	if (startup_config == NULL) {
 		/* and use default if not specified */
-		config_file = DEFAULT_CONFIG_FILE;
-		MSG_NOTICE(msg_module, "Using default configuration file: %s", config_file);
+		startup_config = DEFAULT_STARTUP_CONFIG;
+		MSG_NOTICE(msg_module, "Using default configuration file: %s", startup_config);
+	} else if (access(startup_config, R_OK) == -1) {
```

Comment:
I agree that it would be more elegant to use `config_open_xml()` here. One way to implement it is to let `config_init` also try to parse the internal config and IE list. However, it is also strange to let it parse and then not use it, since the IE list is actually not used by the IPFIXcol base.

The original problem I tried to fix is that I actually want to know immediately when starting IPFIXcol whether my parameters are correct. The current implementation only reports a missing IE list when the first flow records are written to disk. What's your opinion on this?